utils/SwiftIntTypes.py

```diff
@@ -90,16 +90,16 @@ def numeric_type_names_Macintosh_only():
 
 def all_integer_binary_operator_names():
     return ['<<', '>>', '&*', '&', '&+', '&-', '|', '^']
-    
```

Comment:
Ah, didn't see #275. I'll close this pull request to save the headache.

However in the future I think it'd be nice for all the trailing whitespace to be removed. :P